config/routes.rb

```diff
@@ -9,7 +9,9 @@
     namespace :admin do
       resources :products
       resources :users
-      resources :orders
+      resources :orders do
+        resources :order_histories
```

Comment:
```suggestion
        resources :order_histories, only: %i(index)
```
Mình nên defind cụ thể những method có trong recources được sử dụng luôn.
Thứ nhất là để tránh bị thừa, thứ 2 là sẽ tường minh hơn